.github/workflows/ci.yml

```diff
@@ -156,6 +156,36 @@ jobs:
             
             core.summary.addRaw(summary).write();
 
+  golangci_lint_verify:
+    name: "golangci-lint-verify"
+    runs-on: ubuntu-latest
+    permissions:
+      # Required: allow read access to the content for analysis.
+      contents: read
+      # Optional: allow write access to checks to allow the action to annotate code in the PR.
+      checks: write
+    steps:
+      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5
+        with:
+          repository: ${{ env.REPO }}
+          ref: ${{ env.REF }}
+          sparse-checkout: |
+            .github/config/golangci.yml
+            .env
+      - name: Install Task
+        uses: arduino/setup-task@b91d5d2c96a56797b48ac1e0e89220bf64044611 # v2
+        with:
+          version: 3.x
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Read tool versions
+        id: versions
+        run: |
+          source .env
+          echo "golangci_lint=${GOLANGCI_LINT_VERSION}" >> $GITHUB_OUTPUT
+      - name: Verify the golangci-lint config
+        run:
+          task tools:golangci-lint/verify
+
   golangci_lint:
     name: "golangci-lint"
     needs: discover_modules
```

Comment:
i think we should add ` golangci_lint_verify` as a requirement for the matrix to run. wdyt?

Reply:
Sure 👍  Good idea.

Reply:
you are on 🔥 brother